Kudu.Services/ServiceHookHandlers/VSOHandler.cs

```diff
@@ -45,7 +45,7 @@ protected virtual GitDeploymentInfo GetDeploymentInfo(HttpRequestBase request, J
 
             // even it is empty password we need to explicitly say so (with colon).
             // without colon, LibGit2Sharp is not working
-            Uri remoteUrl = new Uri(_settings.GetValue("repoUrl"));
+            Uri remoteUrl = new Uri(_settings.GetValue("RepoUrl"));
```

Comment:
Long story short, there is existing constant in Azure and we should simply reuse that.